README.md

```diff
@@ -1,2 +1,170 @@
+
+You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+
+
+### `npm test`
+
+Launches the test runner in the interactive watch mode for React.\
+
+## Run Test for Linters
+
+For tracking linter errors locally you need to follow these steps:
+
+1. For tracking linter errors in HTML files run:
+```bash 
+npx hint .
+```
+
+2. For tracking linter errors in CSS or SASS files run:
+
+```bash
+npx stylelint "\*_/_.{css,scss}"
+```
+
+3. For tracking linter errors in JavaScript files run:
+
+```bash
+npx eslint .
+```
+
+## Authors
+
+👤 **Yetemegn Lemma**
+
+[GitHub](https://github.com/yetemegn-telaye) | [Twitter](https://twitter.com/Yetemegn19) | [LinkedIn](linkedin.com/in/yetemegn-lemma)
+
+👤 **Chukwuma Paul Mosanya**
+
+[GitHub](https://github.com/blase147) | [Twitter](https://twitter.com/ChukwumaMosanya) | [LinkedIn](https://www.linkedin.com/in/chukwuma-mosanya)
+
+👤 **Ayodeji Abidemi**
+
+[GitHub](https://github.com/demix007) | [Twitter](https://twitter.com/Dat_dope_demix) | [LinkedIn](http://linkedin.com/in/ayodejiabidemi)
+
+
+## ⭐️ Show your support <a name="support"></a>
+Give a ⭐️ if you like this project!
+
+## 🙏 🙏 Acknowledgments <a name="acknowledgements"></a>
+- I would like to thank [@microverseinc](https://github.com/microverseinc) for the templates and insights 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### 📝 License <a name="license"></a>
+> Copyright 2021,   [Yetemegn Lemma, Chukwuma Paul Mosanya, Ayodeji Abidemi]
+
+> Permission is hereby granted, free of charge, to any person obtaining a copy of this [Space Travelers Hub] and associated documentation files, to deal in the [Space Travelers Hub] without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the [Space Travelers Hub], and to permit persons to whom the [Space Travelers Hub] is furnished to do so, subject to the following conditions:
+
+> The above copyright notice and this permission notice shall be included in all copies or substantial portions of the [Space Travelers Hub].
+
+> THE [Space Travelers Hub] IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE [Space Travelers Hub] OR THE USE OR OTHER DEALINGS IN THE [Space Travelers Hub].
+
```

Comment:
- [ ] I apologize to you on behalf of the previous reviewer. She didn't clearly tell you what you should do. You are not supposed to add license file in your readme but you are supposed to add a link to your LICENSE file which you already have in your repository 😊

### So this section should look like this 🟢

```
### 📝 License <a name="license"></a>

This project is [MIT](./LICENSE) licensed.
```